include/z64.h

```diff
@@ -82,15 +82,13 @@ typedef struct{
     /* 0x04 */ MtxF mf;
 } HorseStruct;
 
-// Game Info aka. Static Context (dbg ram start: 80210A10)
```

Comment:
I feel like keeping a mention of "Static Context" helps since it has been used a lot in the past, but I don't really mind either way

Reply:
Is the `GameInfo` name itself definitive?

The debug string we have with `game_info` is for `gSaveContext` (which imo would be a better fit for "static context" btw) (`osSyncPrintf("\ngame_info.mode=[%d] restart_flag", ((void)0, gSaveContext.respawnFlag));`)

If I was to suggest renaming `GameInfo`, I'd probably pick `RegsContext` or something like that

Reply:
It's not supposed to be definitive, but it hasn't been important to rename since you don't actually refer to it when using regs. Right now I would probably suggest something like `GlobalRegs` or `GlobalRegsContext`, but I haven't put much thought into it

Reply:
regarding actually keeping the comment, I feel like in general we have been just hard committing to new names of things (for example, removing mentions of global context)
it may be confusing for people not used to the new names yet, but itll be better for cleanliness and consistency overall

as for what to actually call gameinfo/static context, i dont have too much of an opinion. naming it after regs would be good

Reply:
I'm going to leave "static context" out and renaming GameInfo to another PR